Source/Core/Core/PowerPC/Jit64/Jit_FloatingPoint.cpp

```diff
@@ -560,6 +560,7 @@ void Jit64::fcmpx(UGeckoInstruction inst)
 {
 	INSTRUCTION_START
 	JITDISABLE(bJITFloatingPointOff);
+	FALLBACK_IF(inst.Rc);
```

Comment:
No, this X is for fcmpo vs fcmpu https://github.com/dolphin-emu/dolphin/blob/master/Source/Core/Core/PowerPC/Interpreter/Interpreter_Tables.cpp#L323